docs/styling/all-style-and-layout-props.md

```diff
@@ -0,0 +1,225 @@
+# All Style and Layout Props
+
+```python exec
+import reflex as rx
+
+props = {
+    "align": {
+        "description": "in a flex, it controls the alignment of items on the cross axis and in a grid layout, it controls the alignment of items on the block axis within their grid area (equivalent to align_items)",
```

Comment:
The first word in each of the descriptions should be capitalized

Reply:
updated